.changesets/config_apollo_telemetry_config_rename.md

```diff
@@ -0,0 +1,10 @@
+###  ([#5807](https://github.com/apollographql/router/pull/5807))
+
+All known issues related to the new Apollo usage report generation have been resolved so we are renaming some experimental options to be non-experimental.
+* `experimental_apollo_metrics_generation_mode` is now `apollo_metrics_generation_mode`
```

Comment:
```suggestion
* `experimental_apollo_metrics_generation_mode` is removed
```

Reply:
Oops thanks, this must have been a copy-paste error. I had something else to put here.